src/queries/getData.js

```diff
@@ -0,0 +1,15 @@
+const dbConnection = require("../database/db_connection.js");
+const get4painting = cb => {
+  dbConnection.query(
+    `select id,name,price,img from paintaings order By price desc limit 4;`,
```

Comment:
Keep consistent here e.g. if you're using lowercase please write 'by'. You should also add spaces between the inputs e.g id, name, price